github-pr-notifier/config/definitions/github-pr-notifier.yml

```diff
@@ -0,0 +1,24 @@
+- job:
+    name: github-pr-notifier
+    description: "Update Github PRs *on failure*"
+    node: x86_64
+    project-type: freestyle
+    display-name: 'github-pr-notifier'
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 25
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+    block-downstream: false
+    block-upstream: false
+    concurrent: true
+
+    wrappers:
+      - inject-passwords:
+          global: true
+          mask-password-params: true
+
+    builders:
+      - shell:
+          !include-raw-escape:
```

Comment:
I think ``!include-raw`` would work fine here because this job isn't a ``job-template``. Does this pass everything you need correctly?

Reply:
I did try, and didn't like it because we use braces elsewhere in the Python script which jjb thinks it needs to translate. It didn't work until I was escaping it here